docs/user-guide/src/capm3/node_reuse.md

```diff
@@ -0,0 +1,49 @@
+# Node Reuse
+
+This feature brings a possibility of re-using the same BaremetalHosts (referred to as a host later)
+during deprovisioning and provisioning mainly as part of the rolling upgrade process in the cluster.
+
+## Importance of scale-in strategy
+
+The logic behind the reusing of the hosts, solely relies on the **scale-in** upgrade strategy utilized by
+Cluster API objects, namely [KubeadmControlPlane](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/proposals/20191017-kubeadm-based-control-plane.md#rolling-update-strategy) and MachineDeployment, where during the upgrade process, the machines owned (control-plane or worker) are removed one-by-one before creating new ones (delete-create method).
```

Comment:
Can you please break this long sentence? It would be easier for user to understand.